### PR TITLE
Added Buzzer Beater to demos

### PR DESCRIPTION
**Pull request by Jousboxx**
It would make me veryy happy if you merged this :)))

### CODE REVIEW

**Comment by tresf** [comment]
@Jousboxx please update [`LICENSE.TXT`](https://github.com/Jousboxx/lmms/blob/master/data/projects/demos/LICENSES.TXT) and we'll merge.

Note, moving forward, we will be [stripping all non-CC0 projects](https://github.com/LMMS/lmms/issues/1273#issuecomment-62189752), so this track will be removed in a future version of LMMS if it cannot be released CC0.

Developers, once ready, this PR should be merged against `stable-1.2` branch (or cherry-picked against 1.2, accordingly).

**Comment by tresf** [comment]
.. and a pic, because everyone loves pics... 
screen shot 2017-04-02 at 1 28 50 pm

**Comment by Jousboxx** [comment]
Ok, I updated LICENSES.TXT. I put in BY-SA similar to most of the other projects. I am no lawyer but a quick google shows that CC0 is no rights reserved. I would be ok with that, but BY-SA would be preffered.

File: data/projects/demos/LICENSES.TXT [hunk 1]
```diff
@@ -85,3 +85,7 @@
 * unfa-Spoken.mmpz
   - CC (by-nc)
   - https://lmms.io/lsp/index.php?action=show&file=4929
+  
+  * Jousboxx-BuzzerBeater.mmpz
```
Comment (tresf) [inline data/projects/demos/LICENSES.TXT]: Can you please bump that asterisk in a notch? :)

**Comment by tresf** [comment]
> CC0 is no rights reserved. I would be ok with that, but BY-SA would be preferred.

We will likely remove our demos and start from scratch.  Most musicians don't want their best works going out CC0, and this is OK, but it's problematic to ship a DAW with samples that have restrictions, so it's a long term goal to clean this up.

We may re-ask when that time comes but chances are we will just start from a clean slate. 👍

**Comment by Jousboxx** [comment]
I'm OK with this going out as CC0. This song isn't making me any money, and I will make much better songs in the future so this isn't like giving up the rights to my big hit or anything.

Updated asterisk.

File: data/projects/demos/LICENSES.TXT [hunk 2]
```diff
@@ -86,6 +86,6 @@
   - CC (by-nc)
   - https://lmms.io/lsp/index.php?action=show&file=4929
   
-  * Jousboxx-BuzzerBeater.mmpz
+ * Jousboxx-BuzzerBeater.mmpz
```
Comment (tresf) [inline data/projects/demos/LICENSES.TXT]: @Jousboxx one more space to the left please.

**Comment by Jousboxx** [comment]
Done

**Comment by tresf** [comment]
> I'm OK with this going out as CC0

We'll try our best to remember this when we clean our bundled content, thanks.

> Done

Thanks, merging.

**Comment by Jousboxx** [comment]
Awesome!! :D

**Comment by tresf** [comment]
Cherry-picked to `stable-1.2` as well via 32f07ac.

**Comment by jasp00** [comment]
Which CC version does lmms.io advertise?

**Comment by tresf** [comment]
> Which CC version does lmms.io advertise?

None currently.  The long term goal is `CC0`

**Comment by jasp00** [comment]
@Jousboxx, is CC BY-SA [4.0](https://creativecommons.org/licenses/by-sa/4.0/) fine for this demo?

**Comment by eagles051387** [comment]
Regarding the licensing for bundled content. Couldn't we start a wiki
page that will keep track of new stuff that is going in under the
correct license?

---
Regards, 

Jonathan Aquilina 

On 2017-04-08 21:31, Tres Finocchiaro wrote:

>> I'm OK with this going out as CC0
> 
> We'll try our best to remember this when we clean our bundled content, thanks.
> 
>> Done
> 
> Thanks, merging. 
> 
> --
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub [1], or mute the thread [2].
 

Links:
------
[1] https://github.com/LMMS/lmms/pull/3482#issuecomment-292740370
[2]

**Comment by tresf** [comment]
We should really make an effort to leverage the LSP for future inclusions with some recommendations and some English explanations.

I'm not sure how a wiki would help.

**Comment by Jousboxx** [comment]
@jasp00 sure, CC BY-SA 4.0 works.

**Comment by zonkmachine** [comment]
> @jasp00 sure, CC BY-SA 4.0 works.

Cool! I'm cleaning up some of the licenses right now and will update this. Thanks!

**Pull request merged**